File: DeltaApp/DeltaApp/app/src/main/java/br/unip/hermesbot/model/Chamado.kt
package br.unip.hermesbot.model

data class Chamado(
    val chamadoId: Int = 0,
    val titulo: String,
    val descricao: String,
    val status: String,
    val prioridade: String,
    val dataAbertura: String,
    val dataFechamento: String? = null,
    val usuarioSolicitanteId: Int?,
    val tecnicoResponsavelId: Int?,
    val categoriaId: Int?
)

File: DeltaAPI/DeltaAPI/Models/Chamado.cs
namespace DeltaAPI.Models
{
    public class Chamado
    {
        public int ChamadoId { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public string Status { get; set; }
        public string Prioridade { get; set; }
        public DateTime DataAbertura { get; set; }
        public DateTime? DataFechamento { get; set; }
        public int? UsuarioSolicitanteId { get; set; }
        public int? TecnicoResponsavelId { get; set; }
        public int? CategoriaId { get; set; }
    }
}
